README.md

```diff
@@ -45,9 +45,11 @@ This version builds using:
 
    For Mac/Linux:
    ```shell
-   docker run -v $(pwd)/output:/build/compiled m8c-knulli
+   sudo docker run -v $(pwd)/output:/build/compiled m8c-knulli
+   ```
+   ```shell
+   sudo chown -R $(whoami):$(whoami) output
```

Comment:
This command won't work on macOS. I suggest separating the instructions for Mac and Linx